packages/cms-lib/lib/walk.js

```diff
@@ -7,10 +7,13 @@ function walk(dir) {
       files.map(file => {
         return new Promise((resolve, reject) => {
           const filepath = path.join(dir, file);
-          fs.stat(filepath, (error, stats) => {
+          fs.lstat(filepath, (error, stats) => {
             if (error) {
               reject(error);
             }
+            if (stats.isSymbolicLink()) {
+              resolve(filepath);
```

Comment:
It looks like this only handles file links and will result in the linked file being uploaded. I am not sure that is what we want.

Reply:
added code to reject symlinks and also catch and log errors during walk process -- open to ideas for improving!